website/src/components/collapsible/index.js

```diff
@@ -12,10 +12,10 @@ function Collapsible({children, header, expand}) {
   return (
     <>
         <h2 onClick={toggleExpanded}>
-            { expanded ?  '➖'  : '➕' }
             {header}
+            { expanded ?  ' -'  : ' +' }
         </h2>
-        { expanded ? children : <span>Click to expand</span> }
+        { expanded ? children : <span>{ description }</span> }
```

Comment:
I like this, seems more informative than just having "Click to expand". One thing, i think we should check if `description` was passed in before trying to add it.
```suggestion
        { expanded ? children : <span>{ description ? description : "Click to expand" }</span> }
```